sormas-api/src/main/resources/strings.properties

```diff
@@ -991,7 +1000,10 @@ infoNoAccessToPersonEntities = You don't have the necessary user rights to view
 infoDashboardFinalLaboratoryResult = When a case has multiple samples, only the final laboratory result of the sample with the latest sample collection date is considered.
 infoBulkProcess = %d selected entries are currently being processed. This may take a while. The progress will be updated for each 20 entries that have been processed.
 infoBulkProcessFinished = <b>All selected entries have been processed!</b><br/>You can now close this window.
-infoBulkProcessFinishedWithSkips = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction or already archived.
+infoBulkProcessFinishedWithIneligibleItems = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, because they were not eligible.
```

Comment:
I find the word "eligible" to be very vague in this context. Can't we be more precise and tell the user potential reasons why some of the entries could not be processed (as we do in e.g. the ...FinishedWithSkips string)? What exactly does "not eligible" mean in this context?

Reply:
The exact reason  ( for < 40 entities) in bulk progress bar in bulk progress bar why some entities will not be processed we've agreed to be handled in https://github.com/SORMAS-Foundation/SORMAS-Project/issues/12257.

@adinaflorea9 agreed that for this ticket for the bulk progress bar's content would be ok a general message if the deletion is stopped for these items - without the specific reasons of the problem since this will be handled in https://github.com/SORMAS-Foundation/SORMAS-Project/issues/12257 but if you want to change this, please let me know.  For less than 40 selected items also appears that the events with event participants can not be deleted.
I'll explain below also what scenarios we have till now.

/**********************************************************************************************************************/
Right now this message with the ineligible items looks like this:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/bbf808c8-457a-424e-a128-c37b58317ff8). These 'ineligible' item will be prefiltered since because of some specific reason for entity - bulk action should not be processed at all. 

I've used eligible in order to be able to use this message for all the entities and bulk processes where will be necessary ( I do not know for archive/dearchive/edit and other bulk processes what will be considered ineligible - I still have to check them here https://github.com/SORMAS-Foundation/SORMAS-Project/issues/12257 when I'll work on the ticket but for Delete ( Soft and Permanent) and Restore the following ineligible scenarios we'll have:

1)If the selected events contains events with event participants the items with event participants should not be deleted and a message should appear to the user ( as it was also in the past before the bulk progress bar).
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/512f392f-114a-4cf7-bb24-494730cc9905)   -> This part is implemented in this ticket for < 40 selected entities the reason of ineligible is also shown, > 40 selected entities only the fact there are ineligible items will be shown and those will not be deleted

2) In case of external messages the ineligible items would be the messages with "PROCESSED" status.
In the past we had a functionality which I've preserved in this ticket too:
If we select a number of external messages and at least 1 is PROCESSED ( ineligible for deletion), a message will be displayed to the user and the whole deletion ( for all the entities selected) will be rolledback.
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/3452c3ac-2e4d-402e-b934-a4f41b4566a4)
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/49528e48-8845-4fac-8b46-3cdc464a8e86)

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/0f2fea90-ce32-4ca8-8a38-3dd47284aa01)


In https://github.com/SORMAS-Foundation/SORMAS-Project/issues/12257 I also thought to prefilter the cases/events shared with survnet if there is no communication with Survnet in order to show the entities which due to Survnet problems were not processed.

Other ineligible scenarios can appear from other bulk actions ( archive/dearchive/edit/ other bulk actions), I still have to check all the possibilites.
For each controller and bulk process we would know what would mean ineligible  for instance:
-  events with event partcipants can not be deleted
-   cases without archive rights can not be archived ( for this we have a future ticket to not show the archive at all but is not solved yet and is also blocking the bulk progress) and so on.

Please let me know how would you want to proceed after you read this comment :)

Reply:
I'm fine with leaving it as it is :)